syncer/syncer.go

```diff
@@ -1403,14 +1407,16 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 		s.wg.Add(1)
 		name := queueBucketName(i)
 		s.queueBucketMapping = append(s.queueBucketMapping, name)
-		s.workerLagMap[dmlWorkerLagKey(i)] = atomic.NewInt64(0)
+		workerLagKey := dmlWorkerLagKey(i)
+		s.workerLagMap[workerLagKey] = atomic.NewInt64(0)
 		go func(i int, name string) {
 			ctx2, cancel := context.WithCancel(ctx)
 			ctctx := s.tctx.WithContext(ctx2)
-			s.syncDML(ctctx, name, s.toDBConns[i], s.jobs[i], dmlWorkerLagKey(i))
+			s.syncDML(ctctx, name, s.toDBConns[i], s.jobs[i], workerLagKey)
```

Comment:
seems a wrong usage of closure, pass `workerLagKey` like `i, name`

Reply:
got it...... we do not need workerLagMapIniting